tutorials/Basics/Methods-and-Parameters-for-Linking.ipynb

```diff
@@ -0,0 +1,1455 @@
+       "       1680000000000, 1740000000000, 1800000000000, 1860000000000,\n",
+       "       1920000000000, 1980000000000, 2040000000000, 2100000000000,\n",
+       "       2160000000000, 2220000000000, 2280000000000, 2340000000000,\n",
+       "       2400000000000,             0,   60000000000, 2460000000000,\n",
+       "        120000000000, 2520000000000,  180000000000, 2580000000000,\n",
+       "        240000000000, 2640000000000,  300000000000, 2700000000000,\n",
+       "       2760000000000,  360000000000, 2820000000000,  420000000000,\n",
+       "        480000000000, 2880000000000,  540000000000, 2940000000000,\n",
+       "        600000000000,             0, 3000000000000,  660000000000,\n",
+       "         60000000000, 3060000000000,  120000000000, 3120000000000,\n",
+       "        720000000000,  780000000000,  180000000000, 3180000000000,\n",
+       "        840000000000,  240000000000, 3240000000000,  900000000000,\n",
+       "        300000000000, 3300000000000, 3360000000000,  960000000000,\n",
+       "        360000000000, 3420000000000,  420000000000, 1020000000000,\n",
+       "       1080000000000,  480000000000, 3480000000000, 1140000000000,\n",
+       "        540000000000, 3540000000000, 1200000000000,  600000000000,\n",
+       "       1260000000000,  660000000000, 1320000000000,  720000000000,\n",
+       "        780000000000, 1380000000000, 1440000000000,  840000000000,\n",
+       "       1500000000000,  900000000000, 1560000000000,  960000000000,\n",
+       "       1620000000000, 1020000000000, 1680000000000, 1080000000000,\n",
+       "       1740000000000, 1140000000000], dtype=&#x27;timedelta64[ns]&#x27;)</pre></div></li></ul></div></li><li class='xr-section-item'><input id='section-91b9e6f1-e925-4b43-b03d-acb5e242ea82' class='xr-section-summary-in' type='checkbox' disabled ><label for='section-91b9e6f1-e925-4b43-b03d-acb5e242ea82' class='xr-section-summary'  title='Expand/collapse section'>Attributes: <span>(0)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><dl class='xr-attrs'></dl></div></li></ul></div></div>"
+      ],
+      "text/plain": [
+       "<xarray.Dataset>\n",
+       "Dimensions:    (index: 110)\n",
+       "Coordinates:\n",
+       "  * index      (index) int64 0 1 2 3 4 5 6 7 ... 102 103 104 105 106 107 108 109\n",
+       "Data variables:\n",
+       "    cell       (index) float64 1.0 1.0 1.0 1.0 1.0 1.0 ... 3.0 2.0 3.0 2.0 3.0\n",
+       "    time_cell  (index) timedelta64[ns] 00:00:00 00:01:00 ... 00:29:00 00:19:00"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "track[[\"cell\", \"time_cell\"]]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9c3ada63",
+   "metadata": {},
+   "source": [
+    "Since we know that this dataset contains 3 features, whe can visualize the found tracks with masks created from the *cell* variable:"
```

Comment:
```suggestion
    "Since we know that this dataset contains 3 features, we can visualize the found tracks with masks created from the *cell* variable:"
```